tests/test_things.py

```diff
@@ -222,6 +220,17 @@ def test_last(self):
     def test_tasks(self):
         count = things.tasks(status="completed", last="100y", count_only=True)
         self.assertEqual(count, 10)
+
+        # get task by uuid
+        count = things.tasks(uuid="5pUx6PESj3ctFYbgth1PXY", count_only=True)
+        self.assertEqual(count, 1)
+
+        count = things.tasks(uuid="invalid_uuid", count_only=True)
+        self.assertEqual(count, 0)
+
+        with self.assertRaises(ValueError):
+            things.tasks(uuid="invalid_uuid")
```

Comment:
Gut feeling: that `things.tasks(uuid="invalid_uuid")` throws an error and `things.tasks(uuid="invalid_uuid", count_only=True)` doesn't feel consistent.

Reply:
I do not have a strong opinion here.

My intuition was:
1. `things.tasks(uuid="invalid_uuid")` says "please give me the task with the uuid "invalid_uuid". Response: Sorry, `ValueError`, no such uuid is found.
2. `things.tasks(uuid="invalid_uuid", count_only=True)` says "please count how many tasks have the uuid "invalid_uuid". Response: 0.

What do you think?

Reply:
At the same time `task = things.get(uuid="invalid_uuid")` says `None` to indicate that no such uuid is found, isn't it?

Reply:
Correct. This is modelled after the behavior of Python's `dict.get`. I don't have a strong opinion here either, so happy to change. It feels consistent to me.